src/Build/BackEnd/Components/Logging/ProjectLoggingContext.cs

```diff
@@ -135,7 +135,12 @@ private ProjectLoggingContext(NodeLoggingContext nodeLoggingContext, int submiss
                 properties,
                 items,
                 evaluationId);
-            LoggingService.LogComment(this.BuildEventContext, MessageImportance.Low, "ToolsVersionInEffectForBuild", toolsVersion);
+
+            // No need to log a redundant message in the common case
+            if (toolsVersion != "Current")
```

Comment:
Was wondering if this should have a change wave, since we are removing something user-visible, hence something they could care about, but eh.